Model/Model.tex

```diff
@@ -839,11 +849,11 @@ \subsection{Global variables}
 \hint{Writing to the Transcript is slow, especially when the transcript window is open. 
 So, if you experience some sluggishness and are writing to the Transcript, think about collapsing it.}
 
-\subsubsection{Other useful global variables}
+Transcript outputs are mostly used for debugging and experimenting purposes where code quality is not the most important thing.
+If you want to use a transcript in a real application, you should use \ct{Project current transcript} instead, which makes sure that the transcript you are using actually matches your current \sq environment.
 
-\begin{itemize}
-\item
-\ct{Smalltalk} is the instance of \ct{SystemDictionary} that defines all of the globals\,---\,including \ct{Smalltalk} itself.   
+Another useful global variable is \ct{Smalltalk} which holds the instance of \ct{SystemDictionary} that defines all of the globals\,---\,including \ct{Smalltalk} itself.
```

Comment:
You are obviously more up-to-date but I thought that the direct use of `Smalltalk` was also discouraged?

Reply:
Not sure about this, at least it has more than 1000 senders in my image. @marceltaeumel Sorry for pinging, is it best practice to use `Smalltalk` directly or `SmalltalkImage current` instead?

Reply:
wow, ok nevermind :) I guess it depends on whether one wants to access information on the execution environment (e.g. `Smalltalk isBigEndian`) or whether one wants to access the environment.